src/extension/inpage-script/index.js

```diff
@@ -13,12 +13,7 @@ function init() {
   window.nostr = new NostrProvider();
   window.webbtc = new WebBTCProvider();
   window.webln = new WebLNProvider();
-  window.alby = new AlbyProvider(
```

Comment:
does this work? 

```js
const liquid = new LiquidProvider();
const nostr = new NostrProvider();
const webbtc = new WebBTCProvider();
const webln = new WebLNProvider();

window.liquid = liquid;
window.nostr = nostr;
window.webbtc = webbtc
window.webln = webln;

const alby = new AlbyProvider(liquid, nostr, webbtc, webln);
window.alby = alby;
```

Reply:
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/26054e4c-c33b-44d2-8e24-0a0782a87ff8)

Reply:
unfortunately no